Packs/CortexAttackSurfaceManagement/Scripts/SnmpDetection/SnmpDetection.yml

```diff
@@ -0,0 +1,25 @@
+args:
```

Comment:
add outputs to the yml file

Reply:
Added outputs in the yml [file](https://github.com/demisto/content/blob/9e1f958b137b2552024078e954b6116fea5458d9/Packs/CortexAttackSurfaceManagement/Scripts/SnmpDetection/SnmpDetection.yml#L16)